packages/cardano-graphql/src/DataProjection/util.ts

```diff
@@ -21,22 +28,22 @@ export const mergedQuery = async (
       };
     });
 
-// export const mergedPreProcessingResults = async (
-//   blockHandlers: BlockHandler[],
-//   ctx: RollForwardContext,
-//   queryResult: any
-// ): Promise<any> =>
-//   blockHandlers.map((handler) =>
-//     handler.process
-//       ? { func: handler.process(ctx, queryResult), handler: handler.id }
-//       : { func: noOp(), id: handler.id }
-//   );
+export const mergedProcessingResults = async (
+  blockHandlers: BlockHandler[],
+  ctx: RollForwardContext,
+  mergedQueryResults: any
+): Promise<CombinedProcessingResult[]> =>
+  blockHandlers.map((handler) =>
+    handler.process
+      ? { func: handler.process({ ctx, queryResult: mergedQueryResults }), id: handler.id }
```

Comment:
How about awaiting processing results here? Would need to make sure handlers are processed in parallel though (Promise.all).